rtos/source/Thread.cpp

```diff
@@ -19,6 +19,8 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
+#include <stdlib.h>
```

Comment:
Missed that you don't need this any more - doesn't really matter.

Reply:
Just done it. :+1: